reflex/constants/installer.py

```diff
@@ -95,8 +95,8 @@ class Commands(SimpleNamespace):
         """The commands to define in package.json."""
 
         DEV = "next dev"
-        EXPORT = "next build && next export -o _static"
-        EXPORT_SITEMAP = "next build && next-sitemap && next export -o _static"
+        EXPORT = "next build"
+        EXPORT_SITEMAP = "next-sitemap && next build"
```

Comment:
Seems like this change isn't working.

On the `main` branch if you do `reflex run --env prod` on reflex-web, you can navigate to localhost:3000/sitemap.xml and see the sitemap. After this PR, that page returns a 404.